src/Stratis.Bitcoin.Features.Wallet/Wallet.cs

```diff
@@ -728,12 +728,29 @@ public IEnumerable<UnspentOutputReference> GetSpendableTransactions(int currentC
 
                     if (confirmationCount >= confirmations)
                     {
-                        yield return new UnspentOutputReference
+                        if (includeImmature || (!(transactionData.IsCoinBase ?? false) && !(transactionData.IsCoinStake ?? false)))
```

Comment:
What does it mean if transaction data have null for Coinbase or coinstake flags?